index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+    <header>
+        <span class="logo-placeholder">
+            Kenny Salazar
+        </span>
```

Comment:
This will act as your logo in the future so to meet the requirements and also for [semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) reasons, please change the tag to an `a` tag 🤓